docs/ki-harbor.md

```diff
@@ -63,3 +63,17 @@ Exit psql
 ```
 
 Try to login once again and observe that you are asked to confirm your username.
+
+## Pod multi-attach error
+
+**Problem**
+
+The Pods for helm registry is
```

Comment:
Not sure about this problem, but it looks like this sentence is not finished.
Other files are fine.